_posts/EternalFootman/2016-04-06-dictionaries-2.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+author: EternalFootman
+title: "Kit's Dictionary Exercises 2"
+---
+
+<iframe src="https://trinket.io/embed/python3/5e3fbee7f2" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+I understand that exercise was very simple, but that's why I found it so frustrating; when it's just a matter of not having the correct punctuation, problem solving skills are not helpful. I knew how to add a single item to the dictionary, but adding things in a loop was not in the book. I often take too long to learn this sort of basic element to a new topic; I suspect that I'm missing it because it's deemed too easy to spend time on, but my progress on the following content suffers because of this.
```

Comment:
I sensed the frustration with this.  I think the frustration was preventing you from makeing the leap from the [loops in the dictionary chapter](http://ils.unc.edu/~eah13/textbook/09-dictionaries.html#histogram) to this problem.  The loops I link to are exactly like what you'd need in this problem, except the count doesn't matter.  In fact the code above would solve this problem.  Since it doesn't care what the value is, a count is totally legit.  You could also get rid of the if statements and set each key equal to `""` or etc.

Try to backtrack and see if you can figure out the missing piece for you.  Perhaps the the counting/histograms we've done before didn't sink in as to what they were doing?